ovs/lib/tests/mdsservice_tests/test_mdsservice.py

```diff
@@ -593,10 +604,20 @@ def test_ensure_safety_of_3(self):
         self._check_reality(configs=configs, loads=loads, vdisks=vdisks, mds_services=mds_services)
 
         # Sub-Test 7: Update 2 primary failure domains (Cannot be identical to secondary failure domains)
-        storagerouter_domains[3].domain = domains[3]
-        storagerouter_domains[6].domain = domains[2]
+        storagerouter_domains[3].domain = domains[3]  # sr 2: primary domain 1 -> 3
+        storagerouter_domains[6].domain = domains[2]  # sr 4: primary domain 1 -> 2
         storagerouter_domains[3].save()
         storagerouter_domains[6].save()
+        # | MDS ID | STORAGEROUTER | VPOOL | PRIMARY FD | SECONDARY FD | CAPACITY |
+        # |    1   |       1       |   1   |     1      |      2       |    10    |
+        # |    2   |       2       |   1   |    [3]     |      2       |    10    |
+        # |    3   |       2       |   1   |    [3]     |      2       |    10    |
+        # |    4   |       3       |   1   |     1      |      -       |    10    |
+        # |    5   |       4       |   1   |     2      |      3       |    10    |
```

Comment:
Shouldn't the primary domain `2` be between brackets here too indicating the change/update?